src/main/resources/application.yml

```diff
@@ -0,0 +1,22 @@
+spring:
+  jpa:
+    hibernate:
+      ddl-auto: create-drop
```

Comment:
tak powinna zachowywać sie nasz aplikacja na produkcji?

Reply:
Oczywiście, że nie. Nie powinno się stosować create-drop. Tutaj zostało to użyte wyłącznie w celach testowych.